components/scream/src/share/atm_process/atmosphere_process.cpp

```diff
@@ -76,6 +77,11 @@ void AtmosphereProcess::run (const int dt) {
     run_precondition_checks();
   }
 
+  // Column-wise mass and energy checks requires the total mass and energy
+  // to be computed directly before the atm process is run, as well and store
+  // the correct timestep for the process.
+  setup_conservation_checks_for_this_process(dt);
```

Comment:
I don't see a conservation-check-specific line down below, so I think you can only do the mass/energy checks if `enable_postcondition_checks=true`, right? We're already using a separate PL entry for mass/energy checks, we could probably store the check separate from the other checks, and manually run it based on the value of `enable_column_conservation_checks`. We don't want to have to enable _all_ checks in order to turn on conservation checks...